app/src/main/java/net/kuama/documentscanner/utils/PointUtils.kt

```diff
@@ -0,0 +1,30 @@
+package net.kuama.documentscanner.utils
+
+import net.kuama.documentscanner.data.Corners
+import org.opencv.core.Point
+import org.opencv.core.Size
+import java.util.*
+import kotlin.Comparator
+
+object PointUtils {
+    fun getSortedCorners(it: Array<Point>, originalSize: Size): Corners {
```

Comment:
```suggestion
    fun getSortedCorners(cornersList: Array<Point>, originalSize: Size): Corners {
```
usually `it` is the parameter inside of lambdas in kotlin